File: capstone_team01_banking/budgetting-microservice/src/main/java/com/axlebank/budgettingmicroservice/models/Transaction.java
package com.axlebank.budgettingmicroservice.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Transaction {


    private int transactionId;

    private double amount;

    private LocalDate dateCreated;

    private TransactionType transactionType;
    private String userType;

    private int userId;
    private int institutionId;
}

File: capstone_team01_banking/BankingApp/src/app/enum/transaction.ts
export interface Transaction {
    transactionId: string;
    amount: number;
    dateCreated: string;
    transactionType: string;
    clientId: number;
    adminId: number;
    transactionMedium: string;
    fromAccountNumber: number;
    toAccountNumber: number;
    emailRecipient: string;
    institutionId: number;
    createdBy: string;
}